resources/views/layouts/base.blade.php

```diff
@@ -37,9 +37,14 @@
     <link href="/css/base.css" rel="stylesheet">
     <link href="/css/toastr.min.css" rel="stylesheet">
     <link href="/css/font-awesome.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/morris.js/0.5.1/morris.css">
 
     <link rel="shortcut icon" href="favicon.ico">
     @yield('css')
+
+    <script src="/js/jquery-1.11.3.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/raphael/2.1.0/raphael-min.js"></script>
```

Comment:
We should download these external JS/CSS libraries locally and access them that way. 
Raphael and Morris specifically could be placed in the CSS & JS sections of the templates re: statistics; we don't need them on `base.blade.php`, because it would unnecessarily slow down the load time.

Also, is there a reason JS, Morris, and Raphael need to be in the `<head>`? Otherwise, we should load them asynchronously at the end of the `<body>` with the other scripts.